run_workflow.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import logging
+import yaml
+
+from typing import Any
+
+from neat_ml.workflow.lib_workflow import (get_path_structure, 
+                                           stage_detect)
+
+log = logging.getLogger(__name__)
+
+def main(config_path: str, steps_str: str) -> None:
+    """
+    Orchestrate selected workflow stages.
+
+    Parameters
+    ----------
+    config_path : str
+        Path to config YAML.
+    steps_str : str
+        Comma separated list of steps. (Currently, only detect)
+    """
+    steps = [s.strip() for s in steps_str.split(",") if s.strip()]
+
+    with open(config_path, "r") as fh:
+        cfg: Any = yaml.safe_load(fh)
+
+    roots: dict[str, Any] = cfg["roots"]
```

Comment:
shouldn't need this type hint either.. only in func sigs; also, doesn't `mypy` only run on the package proper anyway?